src/main/java/uk/gov/pay/directdebit/app/config/EncryptionConfig.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.pay.directdebit.app.config;
+
+import io.dropwizard.Configuration;
+
+public class EncryptionConfig extends Configuration {
+
+    private String encryptDBSalt;
+
+    public String getEncryptDBSalt() {
```

Comment:
As discussed we don't want to use fixed salt with `bcrypt` hash. The salt is randomly generated for each hash and it is embedded in the hash. We can use `BCrypt.gensalt(12)` to generate random salt as described in https://www.mindrot.org/projects/jBCrypt/

Reply:
👍 done, thanks